x-pack/plugins/observability_solution/observability_ai_assistant/server/functions/index.ts

```diff
@@ -84,11 +86,17 @@ export const registerFunctions: ChatRegistrationFunction = async ({
         `
             : ''
         }
+
+        The user is able to change the language in which it wants you to reply in the settings page of the AI Assistant for Observability, which can be found in the ${
```

Comment:
nit
```suggestion
        The user is able to change the language in which they want you to reply on the settings page of the AI Assistant for Observability, which can be found in the ${
```

Reply:
Fixed in https://github.com/elastic/kibana/pull/176444/commits/a1438dd6322b9149cf46f37f891bf9ad10283fd7